src/anyio/_backends/_asyncio.py

```diff
@@ -594,6 +597,29 @@ def _run_in_thread(self, func: Callable, *args):
         return run_in_thread(func, *args)
 
 
+async def getaddrinfo(host: str, port: int, *, family: Union[int, AddressFamily] = 0,
```

Comment:
`socket.getaddrinfo` accepts both `str` and `bytes`; do you want to match that?

Reply:
Yeah – the typeshed signature for that is `Union[bytearray, bytes, Text]` so I should probably add those too. Good catch!